journal/updated/09-chapter-8.adoc

```diff
@@ -422,9 +429,9 @@ To Her Grandchildren.
 [.signed-section-context-open]
 Donnington, Third month, 1804.
 
-Being likely to have a ready conveyance, I am set down as in the depth of poverty;
-but I felt willing to salute my dear grandchildren once more;
-though it will expose my weakness, yet it will manifest my willingness,
+Being likely to soon depart, I have been set down as in the depth of poverty;
+but I felt willing to salute my dear grandchildren once more.
+And though it will expose my weakness, yet it will manifest my willingness,
 and that you are very frequently in my remembrance.
 I am glad that you are so agreeably placed, and wish you to improve in the best things,
 as it seems you are encouraged, and I would hope exampled thereto.
```

Comment:
This sounds kinda old-fashion. Maybe we can change it to say: **as it seems you are encouraged to do both by word and example.**